generator/src/main.rs

```diff
@@ -25,7 +25,7 @@ fn main() {
     args.next().unwrap();
     let mut source = File::open(args.next().map(PathBuf::from).unwrap_or_else(|| {
         Path::new(env!("CARGO_MANIFEST_DIR"))
-            .join("../sys/OpenXR-SDK/specification/registry/xr.xml")
+            .join("xr.xml")
```

Comment:
The commit that adds the new files should be squashed into this one to make it practical to review the changes in the generated output.

Reply:
it is?